homeassistant/components/reolink/binary_sensor.py

```diff
@@ -49,42 +49,42 @@ class ReolinkBinarySensorEntityDescription(
 BINARY_SENSORS = (
     ReolinkBinarySensorEntityDescription(
         key="motion",
-        name="Motion",
+        translation_key="motion",
```

Comment:
```suggestion
```
For binary sensors, sensors, numbers (and soon update) the name of the entity can be done via device classes